specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/examples/Operation/Get.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2017-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": {
+          "deploymentData": {
+            "externalDsmsCertificates": "",
+            "customCloudVerificationKey": "",
+            "customEnvironmentEndpoints": {
+              "customCloudArmEndpoint": "https://management.azure.microsoft.scloud/",
+              "externalDsmsEndpoint": "https://global-dsms.dsms.core.microsoft.scloud/dsms/issuercertificates"
+            }
+          },
+          "signature": ""
+        }
```

Comment:
This is not an RPC-compliant response for a resource GET. Please take a look at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md) for details.

Reply:
Added required fields

Reply:
Looks good now.